platforms/core-runtime/gradle-cli/src/main/java/org/gradle/launcher/cli/DefaultCommandLineActionFactory.java

```diff
@@ -94,8 +94,23 @@ private static BuildClientMetaData clientMetaData() {
         return new DefaultBuildClientMetaData(new GradleLauncherMetaData());
     }
 
-    private static void showUsage(PrintStream out, CommandLineParser parser) {
+    private static void showUsage(PrintStream out, CommandLineParser parser, @Nullable String suggestedTaskName) {
         out.println();
+        out.print("To see more detail about a task, run ");
```

Comment:
🤔  There have been some efforts recently about declattering the Gradle's troubleshooting output. In that lane, I suggest we don't print the line about the task at all, if there were no tasks involved. If I run something like `gradle --someflag`, having the line about running tasks is not helpful to have in the output.

Reply:
What about `gradle --wrapper`?

```
Unknown command-line option '--wrapper'.

To see more detail about a task, run gradle help --task <task>

To see a list of available tasks, run gradle tasks

USAGE: gradle [option...] [task...]
```

Reply:
TBH, I think for a novice user, this `To see a list of available tasks, run gradle tasks` should be helpful.